Command/ReindexCommand.php

```diff
@@ -199,10 +199,22 @@ protected function indexDocuments(string $locale, IndexerInterface $indexer, Out
         $progressBar = new ProgressBar($output, $count);
         $progressBar->setFormat(' %current%/%max% [%bar%] %percent:3s%% %elapsed:6s%/%estimated:-6s% %memory:6s%');
         $progressBar->start();
+        $count = 0;
 
         foreach ($documents as $document) {
             $indexer->index($document);
             $progressBar->advance();
+
+            ++$count;
+
+            if (0 === ($count % 100)) {
```

Comment:
```suggestion
            if ($count !== 0 && 0 === ($count % 100)) {
```

to prevent flushing after the first

Reply:
@Prokyonn no flush on first as ++$count is done before ;)

Reply:
ahh yeah nvm 🙈